opensearch-jobs.yml

```diff
@@ -196,7 +200,11 @@ instance_groups:
         - -Dlog4j2.formatMsgNoLookups=true
         migrate_data_path: true
     release: opensearch
-  - name: archiver_syslog
+  - consumes:
```

Comment:
How is this related to the changes?

Reply:
it got deleted when removed by mistake and is needed